.github/actions/get_repo_name/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Get Repo Name'
+description: 'Gets the repository name that was added to the open-repositories.txt file.'
+outputs:
+  repo-name:
+    value: ${{ steps.get-repo-name.outputs.new_repo }}
+    description: "The name of the repository that was added."
+
+runs:
+  using: 'composite'
+  steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get repo name
+        id: get-repo-name
+        run: |
+          NEW_REPO=$(git diff origin/master...HEAD open-repositories.txt | grep "^+[a-z]" | sed 's/+//')
+          REPO_COUNT=$(echo "$NEW_REPO" | wc -l)
+          if [ $REPO_COUNT > 1 ]; then
+            echo "Only one repository can be added at a time. Please create separate PRs."
+            exit 1
+          elif [ $REPO_COUNT = 0 ]; then
```

Comment:
This is probably valid case - PR could be about removing existing repository from the list, not adding the new one.
We may not allow this by policy (but AFAIK we currently allow this), but then the message should clearly say this.

Reply:
That's a very good point. I made a change to handle no changes separately. Tested it again here https://github.com/dfinity/repositories-open-to-contributions/pull/35 and it successfully skips the compliance checks if a repo was removed:
<img width="306" alt="Screenshot 2023-02-23 at 14 00 54" src="https://user-images.githubusercontent.com/47304080/220913553-8b0ac9ab-f780-4b01-98d5-cdc55a1abc51.png">
Obviously will want to check the policy on this but at least the code can handle it.